pom.xml

```diff
@@ -637,6 +637,11 @@
                 <groupId>com.github.spotbugs</groupId>
                 <artifactId>spotbugs-maven-plugin</artifactId>
                 <version>4.5.2.0</version>
+                <configuration>
+                    <includeFilterFile>dev/spotbugs-include.xml</includeFilterFile>
+                    <xmlOutput>true</xmlOutput>
+                    <failOnError>false</failOnError>
```

Comment:
should it be true?

Reply:
It should be false, otherwise, the following steps of GitHub actions will not execute and there won't be a result of the spotbugs analysis annotations.

Reply:
ok